pkg/services/requests.go

```diff
@@ -179,22 +187,22 @@ func RevokeRequest(ctx context.Context, client *aponoapi.AponoClient, requestID
 func ColoredStatus(request clientapi.AccessRequestClientModel) string {
 	status := request.Status.Status
 	if IsRequestWaitingForHumanApproval(&request) {
-		status = requestWaitingForApprovalStatus
+		status = AccessRequestWaitingForApprovalStatus
 	}
 
-	statusTitle := cases.Title(language.English).String(string(status))
+	statusTitle := cases.Title(language.English).String(status)
 	switch status {
-	case requestWaitingForApprovalStatus:
+	case AccessRequestWaitingForApprovalStatus:
 		return color.HiYellow.Sprint(statusTitle)
-	case clientapi.ACCESSSTATUS_PENDING:
+	case AccessRequestInitStatus, AccessRequestPendingStatus:
```

Comment:
AccessRequestWaitingForApprovalStatus and AccessRequestPendingStatus are same one

Reply:
thats why they have the same color